packages/nimble-components/src/table/components/header/template.ts

```diff
@@ -20,8 +21,8 @@ export const template = html<TableHeader>`
         ${when(x => x.sortDirection === TableColumnSortDirection.descending, html`
             <${iconArrowDownTag} class="sort-indicator" aria-hidden="true"></${iconArrowDownTag}>
```

Comment:
Currently I wasn't planning to add sort labels. From the [HLD](https://github.com/ni/nimble/blob/main/specs/labels-and-localization/README.md?plain=1#L27):

> No accessible label specifically for these icons is currently planned. The primary sorted column is already indicated with `aria-sort="ascending"` or `aria-sort="descending"`. The [ARIA APG's sortable table example](https://www.w3.org/WAI/ARIA/apg/patterns/table/examples/sortable-table/) follows similar logic, and says sort labels are not added to each column header button "to prevent repetitious verbosity that could interfere with understanding of the column titles".

Reply:
excellent, maybe add a comment to that effect saying that avoiding a title on the sort icons is intentional?

Reply:
Added